app/src/main/java/com/example/datastoreapp/activity/MainActivity.kt

```diff
@@ -3,7 +3,7 @@ package com.example.datastoreapp.activity
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.example.datastoreapp.databinding.ActivityMainBinding
-
+// Borrar esta activity y su layout
```

Comment:
This is a WIP? Did you forget to remove this comment?

Reply:
+1